Vwitter/View Controllers/ComposeViewController.m

```diff
@@ -0,0 +1,69 @@
+//
+//  ComposeViewController.m
+//  Vwitter
+//
+//  Created by Christina Li on 7/8/22.
+//
+
+#import "ComposeViewController.h"
+
+@interface ComposeViewController () <UITextViewDelegate>
+@property (weak, nonatomic) IBOutlet UITextView *ventContent;
+@property (weak, nonatomic) NSString *placeholderText;
+
+@end
+
+@implementation ComposeViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+    self.ventContent.delegate = self;
+    self.placeholderText = @"Let it out!";
+    self.ventContent.text = self.placeholderText;
+    self.ventContent.textColor = [UIColor lightGrayColor];
+}
+
+- (IBAction)didTapClose:(id)sender {
+    [self dismissViewControllerAnimated:true completion:nil];
+}
+
+- (void)textViewDidBeginEditing:(UITextView *)textView
+{
+    if ([textView.text isEqualToString:self.placeholderText]) {
+         textView.text = @"";
+         textView.textColor = [UIColor blackColor];
+    }
+    [textView becomeFirstResponder];
+}
+
+- (void)textViewDidEndEditing:(UITextView *)textView
+{
+    if ([textView.text isEqualToString:@""]) {
+        textView.text = self.placeholderText;
+        textView.textColor = [UIColor lightGrayColor];
+    }
+    [textView resignFirstResponder];
+}
+
+- (BOOL)textView:(UITextView *)textView shouldChangeTextInRange:(NSRange)range replacementText:(NSString *)text {
+
+    if([text isEqualToString:@"\n"]) {
```

Comment:
One potential improvement: I was thinking, if you use the "done"/return button on the keyboard to dismiss the keyboard, then this will prevent users from being able to write multiple lines. If you are fine with that, then it's no problem